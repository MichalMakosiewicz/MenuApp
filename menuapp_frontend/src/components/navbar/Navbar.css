.root {
    flex-grow: 1,
}

.menu-button {
    margin-Right: 20px,
}

.button {
    margin-left: 30px;
}

.title {
    flex-grow: 1;
}