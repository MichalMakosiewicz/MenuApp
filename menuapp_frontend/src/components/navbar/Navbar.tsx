import React from 'react';
import AppBar from '@material-ui/core/AppBar';
import Toolbar from '@material-ui/core/Toolbar';
import Typography from '@material-ui/core/Typography';
import IconButton from '@material-ui/core/IconButton';
import MenuIcon from '@material-ui/icons/Menu';
import Button from '@material-ui/core/Button';
import "./Navbar.css";

export function Navbar() {

    return (
        <div className="root">
            <AppBar position="fixed">
                <Toolbar>
                    <Typography variant="h6" className="title">
                        Menu App
                    </Typography>
                    <Button className="button" href="/admin" variant="contained" color="secondary">
                        Go to Admin site
                    </Button>
                </Toolbar>
            </AppBar>
        </div>
    );
}