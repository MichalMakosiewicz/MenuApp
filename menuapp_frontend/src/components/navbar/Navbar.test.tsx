import React from 'react';
import {Navbar} from './Navbar';
import renderer from 'react-test-renderer';

describe('Navbar test', () => {
    it('match the snapshot', () => {
        const tree = renderer.create(<Navbar/>).toJSON();
        expect(tree).toMatchSnapshot()
    })
})