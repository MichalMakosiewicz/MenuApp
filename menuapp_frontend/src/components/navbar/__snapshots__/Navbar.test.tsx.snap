// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar test match the snapshot 1`] = `
<div
  className="root"
>
  <header
    className="MuiPaper-root MuiAppBar-root MuiAppBar-positionFixed MuiAppBar-colorPrimary mui-fixed MuiPaper-elevation4"
  >
    <div
      className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
    >
      <h6
        className="MuiTypography-root title MuiTypography-h6"
      >
        Menu App
      </h6>
      <a
        aria-disabled={false}
        className="MuiButtonBase-root MuiButton-root MuiButton-contained button MuiButton-containedSecondary"
        href="/admin"
        onBlur={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
      >
        <span
          className="MuiButton-label"
        >
          Go to Admin site
        </span>
      </a>
    </div>
  </header>
</div>
`;
